interfaces/sourcegen/src/sourcegen/headers/ctdomain_auto.yaml

```
@@ -19,6 +19,13 @@ recipes:
 - name: nPoints
 - name: componentName
 - name: componentIndex
+  wraps: componentIndex(const char*)
```

Should this be:
```suggestion
  wraps: componentIndex(const string&)
```
given the signature of the actual C++ method?